tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -23,6 +23,7 @@ def format_struct(struct):
 
 def setup_defaults(config):
     config.setdefault('region', 'us-east-1')
+    config.setdefault('region', config['region'])
```

Comment:
from pr comments, its sounds like this was intended to be ses_region instead of just no-op setting region value again.